db/primary_migrate/20230727195406_add_in_person_verification_pending_at_to_profiles.rb

```diff
@@ -0,0 +1,8 @@
+class AddInPersonVerificationPendingAtToProfiles < ActiveRecord::Migration[7.0]
+  disable_ddl_transaction!
+
+  def change
+    add_column :profiles, :in_person_verification_pending_at, :datetime
+    add_index :profiles, :in_person_verification_pending_at, algorithm: :concurrently
```

Comment:
Do we plan to query by this column in the future?

Reply:
Not that much. I'll remove the index. If we do need it we can add it back.